dev/release/setup-centos7.sh

```diff
@@ -0,0 +1,90 @@
+
+#!/bin/sh
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# A script to install dependencies required for release
+# verification Red Hat Enterprise Linux 7 clones in particular
+# on CentOS 7
```

Comment:
```suggestion
# A script to install dependencies required for release
# verification on CentOS 7
```

because this script's name is `setup-centos7.sh`.

Reply:
There are other long lived distributions such as:
- https://puias.math.ias.edu/
- https://scientificlinux.org/
- https://yum.oracle.com/oracle-linux-isos.html
- https://www.cybertrust.co.jp/miracle-linux/7/
- https://www.chinaredflag.cn/product/newproduct?ulink=&tolink=redflag-web/AX7FT
Maybe the script should be called RHEL7 derivatives? Or maybe add a comment it should work on these.

Reply:
If we want to support them, `setup-rhel7-rebuilds.sh` will be better.
But we will not want to support all of them...

Reply:
Renamed script.